src/components/Dashboard/TodoList.vue

```diff
@@ -142,22 +135,20 @@
                     <span>FILING PENDING</span>
                     <span class="vert-pipe"></span>
                     <span v-if="inProcessFiling === item.filingId">PROCESSING...</span>
-                    <span v-else class="d-flex align-center">
-                      <span v-if="isPayMethodOnlineBanking(item)">ONLINE BANKING PAYMENT PENDING</span>
-                      <span v-else>PAYMENT INCOMPLETE</span>
-                    </span>
+                    <span v-else-if="isPayMethodOnlineBanking(item)">ONLINE BANKING PAYMENT PENDING</span>
+                    <span v-else>PAYMENT INCOMPLETE</span>
```

Comment:
Extra classes were not needed so I cleaned up this code.

Reply:
Screenshot (without classes):

![image](https://user-images.githubusercontent.com/10002889/207436732-532fe26e-dd37-43b4-a952-3f233583f643.png)